.github/workflows/build.yml

```diff
@@ -546,7 +545,6 @@ jobs:
             vulkan-devel:p
 
       - name: Install dependencies
-        if: ${{ matrix.sys != 'mingw32' }}
         run: |
           pacboy --noconfirm -S {angleproject,cppwinrt,rst2pdf,rubberband}:p
```

Comment:
This can be merged now to above list of packages. Also build_msys2.sh can be cleaned from `mingw32` references.